packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md

```diff
@@ -0,0 +1,75 @@
+
+  * **Allow partial matches** &mdash; Partial matching occurs when the first and last name of an imported user match those of an existing Okta user, but the username or email address do not.
+
+  * **Confirm matched users** &mdash; Select to automate the confirmation or activation of existing users. If the option isn't selected, matches are confirmed manually.
+
+  * **Confirm new users** &mdash; Select to automate the confirmation or activation of a newly imported user. If this option is selected, you can clear it during import confirmation. Note that this feature doesn't apply for users who already exist in Okta.
+
+* **Profile & Lifecycle Mastering** &mdash; Use this section to allow the current application to profile master Okta users. If enabled, the application appears in the list of profile masters on the Profile mastering page.
+
+  * **Allow `<app>` to master Okta users** &mdash; Enable mastery and determine what happens when a user is deactivated or reactivated in an application. Only the highest priority profile master for that Okta user can deactivate or suspend an Okta user. To verify the highest priority profile master, review the Profile mastering page.
+
+  * **When a user is deactivated in the app** &mdash; Select **Do Nothing** to prevent activity in the application from controlling the user life cycle. This still allows profile master control of attributes and mappings. Otherwise, you can choose between deactivating or suspending the user.
+
+  * **When a user is reactivated in the app** &mdash; Decide if suspended or deactivated Okta users should be reactivated when they have been reactivated in the application.
+
+    >**Note** When a user is reactivated in the app, the user profile must be an exact match to the Okta profile for the reactivation to also occur in Okta. Otherwise, after importing the reactivated users, they appear in **Pending Activation** state.
+
+* **Import Safeguards** &mdash; The import safeguard settings define the maximum percentage of application users in an org that can be unassigned while still allowing the import to proceed. App-level and org-level safeguards are enabled by default and set at 20 percent.
+
+* **Inline Hooks** &mdash; Use this section to add custom logic to the process of importing new users into Okta from an app. You can resolve conflicts in profile attributes and control whether imported users are treated as matches for existing users. To enable an import inline hook, see [Inline hooks](/docs/concepts/inline-hooks/).
+
+* **Okta Attribute Mappings** &mdash; Use this portion of the page to edit attributes and mappings in the Manage user profiles.
```

Comment:
this section of the page?

Reply:
Replaced with text similar to the end of the "To Okta" section:
Edit attributes and mappings through the Profile Editor. See the <GuideLink link="../attribute-mapping/">Attributes and mappings</GuideLink> section in this guide or [Work with Okta user profiles and attributes](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Manage_Profile_Attributes) in the Okta product documentation.